astronomy/solar_system_dynamics_test.cpp

```diff
@@ -216,8 +217,8 @@ class SolarSystemDynamicsTest : public ::testing::Test {
     }
     RigidMotion<ICRS, ParentEquator> const to_parent_equator(
         {ICRS::origin, ParentEquator::origin, rotation->Forget()},
-        /*angular_velocity_of_to_frame=*/{},
-        /*velocity_of_to_frame_origin=*/{});
+        AngularVelocity<ICRS>{},
+        Velocity<ICRS>{});
```

Comment:
Keep the names, it's hepful to see that it's the velocity *of* to frame, not just that it's expressed *in* from frame.